version/2_7.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash -e
 
-echo "================= Installing Python 2.7 ==================="
-# python 2.7 is already installed
-yum install -y python-devel-2.7*
+export PYTHON_2_7_15_VERSION=2.7.15
```

Comment:
This env var name makes no sense at all. Are you really going to _rename_ the env var every time the version changes?